File: java/src/main/java/com/lacombe/promo3/taxi/Arrival.java
package com.lacombe.promo3.taxi;

public class Arrival {
    private final ArrivalHour hour;
    private final Participant participant;

    public Arrival(ArrivalHour hour, Participant participant) {
        this.hour = hour;
        this.participant = participant;
    }

    public ArrivalHour getHour() {
        return hour;
    }


    public String getParticipantName() {
        return participant.getName();
    }
}

File: dotnet/Socrates/TaxiOrganization/Arrival.cs
namespace Socrates.TaxiOrganization
{
    public class Arrival
    {
        private readonly ArrivalHour hour;
        private readonly Participant participant;

        public Arrival(ArrivalHour hour, Participant participant)
        {
            this.hour = hour;
            this.participant = participant;
        }

        public ArrivalHour GetHour()
        {
            return hour;
        }

        public string GetParticipantName()
        {
            return participant.GetName();
        }
    }
}